OpenRA.Game/WorldUtils.cs

```diff
@@ -90,8 +90,7 @@ public static void DoTimed<T>(this IEnumerable<T> e, Action<T> a, string text)
 
 		public static bool AreMutualAllies(Player a, Player b)
```

Comment:
This is only ever used from the KOTH code, and is almost certainly left over from the days where we (incorrectly) supported assymetric relationships.

I won't block this PR over it, but IMO it would be good if a followup could remove this.